main.go

```diff
@@ -80,22 +77,21 @@ func main() {
 			log.Fatal("Please provide a named AWS profile")
 		}
 
-		if *region == "" {
-			regions, err = common.ListEnabledRegions(ctx, profile)
-			common.ExitOnError(err, "Failed to list enabled regions for profile "+*profile)
-		} else {
-			regions = []string{*region}
-		}
+		accountDetails, err := common.GetAccountDetails(ctx, *profile, *region)
+		common.ExitOnError(err, "Failed to get account details")
 
-		for i, r := range regions {
-			fmt.Printf("Region %d: %s\n", i+1, r)
-			vpcutils.FixEc2_2(ctx, *profile, r, *execute)
-			fmt.Printf("----------------------------------------------------\n\n")
-		}
+		ch := make(chan vpcutils.SecurityGroupRuleDetails)
```

Comment:
We are now finding unused security group rules in each region concurrently, and writing the results to a [channel](https://gobyexample.com/channels)